src/main/kotlin/org/hyacinthbots/lilybot/extensions/moderation/TemporaryModeration.kt

```diff
@@ -48,13 +48,16 @@ import org.hyacinthbots.lilybot.extensions.config.ConfigOptions
 import org.hyacinthbots.lilybot.extensions.config.ConfigType
 import org.hyacinthbots.lilybot.utils.baseModerationEmbed
 import org.hyacinthbots.lilybot.utils.botHasChannelPerms
-import org.hyacinthbots.lilybot.utils.configPresent
+import org.hyacinthbots.lilybot.utils.configIsUsable
 import org.hyacinthbots.lilybot.utils.dmNotificationStatusEmbedField
 import org.hyacinthbots.lilybot.utils.getLoggingChannelWithPerms
 import org.hyacinthbots.lilybot.utils.isBotOrModerator
+import org.hyacinthbots.lilybot.utils.requireConfigs
 import java.lang.Integer.min
 import kotlin.time.Duration
 
+// todo many of these commands require integration with public logging
```

Comment:
It was present in all of them that needed it, why was this removed???

Reply:
I didn't remove it, I just moved it above the private logging embed so we can still public log if private logging is broken. And I think there are a few that weren't done but I'll come up with a list later.